docs/runjobs/scheduled-jobs/hero-runs.md

```diff
@@ -41,6 +41,11 @@ through the general contact form, as the "large-scale runs" category.
   something that cannot be done in the normal production queues. As well as
   evidence that you already have it running on LUMI.
 
+## Once the run is granted
+
+Before the actual full machine run, you will be informed about specific settings and small tests to do. The LUMI user support team will be in contact with you about these. 
+Notice that the actual full machine run slots are zero-support sessions, and on the full machine run Sunday your team should be able to manage the run yourselves. 
```

Comment:
Please note that support is not available the days full-machine runs are carried out.

Reply:
This is now changed